### ISSUE TITLE
Invalid config for 'dohome_rgb' at configuration.yaml

### ISSUE THREAD

**Issue by MarcoLeongDev** [mishamyrt/dohome_rgb#2]
I am aware of the [issue](https://github.com/mishamyrt/dohome_rgb/issues/1) and ensure that the syntax is latest for the yaml. Invalid config is still being encountered.

The main error log:
<img width="1066" alt="Screenshot 2024-01-15 at 11 55 45" src="https://github.com/mishamyrt/dohome_rgb/assets/4134457/3a8d7120-e41c-40fd-bb0b-0c244ffd88a5">

Followed by another two log entries:
<img width="477" alt="Screenshot 2024-01-15 at 11 57 10" src="https://github.com/mishamyrt/dohome_rgb/assets/4134457/99622625-1a79-4da1-8fd8-d2ab4c140956">
<img width="613" alt="Screenshot 2024-01-15 at 11 57 16" src="https://github.com/mishamyrt/dohome_rgb/assets/4134457/0a05f90f-1e0b-474b-9186-23a823118c51">

Other info:
- Running on **Home Assistant OS**
- The custom component is downloaded via HACS
- The `config/configuration.yaml` file shows the following, where `7ED0` is the last 4 chars of the MAC address of the device

```
# Loads default set of integrations. Do not remove.
default_config:

# Load frontend themes from the themes folder
frontend:
  themes: !include_dir_merge_named themes

automation: !include automations.yaml
script: !include scripts.yaml
scene: !include scenes.yaml

light living_room:
  - platform: dohome_rgb
    sids:
      - 7ED0
```

Questions:
1. The device is connected but not in pairing model, is that a requirement?
2. From the error point of view looks like it is not because of variable/argument but rather some function/component calls isn't working? Any experience on this type of issue?
3. Let me know which direction I should be looking at to troubleshoot the issue?

I am quite new to custom components, especially the troubleshooting, thank you for your understand and help in advance

**Comment by mishamyrt** [comment]
Hi! Regarding your first question, the device must be connected to wifi and configured. For the others, the configuration seems to be in a broken format:

```yaml
light living_room:
  - platform: dohome_rgb
    sids:
      - 7ED0
```

should be

```yaml
# remove living_room
light:
  - platform: dohome_rgb
    sids:
      - '7ed0' # lowercase is better
```

At the moment I haven't implemented entity naming through the configuration, I'll probably add it later. For now I can only recommend to configure without naming, and then set what via UI.

**Comment by mishamyrt** [comment]
Checked it again and yes, you're right. There is an error in the code, I will fix it

**Comment by mishamyrt** [comment]
Problem fixed in version v0.5.2, update the integration please
